tests/integration/test_appdynamics.py

```diff
@@ -46,3 +56,15 @@ def test_appdynamics_mx7(self):
 
     def test_appdynamics_mx6(self):
         self._test_appdynamics("sample-6.2.0.mda")
+
+    def test_machine_agent_mx9(self):
```

Comment:
why do we need separate methods for test machine_agent? Wouldnt it be better if the test_appdynamics method takes care of testing both java agent and machine agent?